volttrontesting/platform/web/test_certs.py

```diff
@@ -155,6 +155,8 @@ def test_create_signed_cert_files(temp_volttron_home):
     assert existing_cert[0] == certs.cert("test_cert")
 
 
+@pytest.mark.skipif('OpenSSL' not in os.listdir("/home/volttron/git/myvolttron/env/lib/python3.8/site-packages"),
```

Comment:
hmmm...we can't make this assumption about where openssl livs.

Reply:
yea I was a little unsure about this part, do you have a recommendation on how I could check for openssl better?

Reply:
A couple of options to determine if you have a package installed:
- use https://docs.python.org/3.8/library/importlib.html
- attempt to import and set a flag to tell you whether or not it is imported
try:
  import openssl
  HAS_OPENSSL = True
except ImportError:
  HAS_OPENSSL = False